java/Anton-Kozyk/src/com/zoolatech/lecture4/tasks/_2/BinaryTree.java

```diff
@@ -0,0 +1,59 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class BinaryTree {
+    private Node root;
+
+    static class Node {
+        private int value;
+
+        public Node(int value) {
+            this.value = value;
+        }
+
+        Node leftBranch;
```

Comment:
When you read instance variable - you access them directly, when you write values to them - you use setter methods. It's better to use a single approach. Note: this should be considered only regarding nested classes. For other prefer to use getter and setter methods (because of the encapsulation).